pkgs/applications/misc/deco/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, scsh, feh }:
+stdenv.mkDerivation rec {
+  pname = "deco";
+  version = "0.0.1";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "ebzzry";
+    repo = pname;
+    rev = "f58e931ccea359579a9192748d1085fae73a9d47";
+    sha256 = "06zdb0wgxjf1xk7d0lsc82dmx2vz0a7mn1vkn79x0inyfgp4bdql";
+  };
+
+  propagatedBuildInputs = [ scsh feh ];
+
+  buildPhase = ''
```

Comment:
The `buildPhase` is now redundant. Use `dontBuild = true` to make sure that the default build phase isn't being executed, if it causes problems.